Deadline.java

```diff
@@ -0,0 +1,19 @@
+public class Deadline extends Task {
+
+    protected String by;
+
+    public Deadline(String description, String by) {
+        // tasks that need to be done before a specific date/time
```

Comment:
Can consider not repeat in comments information that is already obvious from the code.